### PR TITLE
Fixes for apps in subfolders

### PR DESCRIPTION
**Pull request by jotaelesalinas**
To make pretty URLS work when using an Apache alias:
- Added a `RewriteBase` directive to `public/.htaccess` (commented out
by default)

To make links and forms in the default views work:
- Changed hardcoded absolute URLs for their relative counterparts, using
`url()`.

### CODE REVIEW

**Comment by AtroxDev** [comment]
:+1:

File: resources/views/app.blade.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Laravel</title>
 
-	<link href="/css/app.css" rel="stylesheet">
+	<link href="{{ URL::asset('css/app.css') }}" rel="stylesheet">
```
Comment (AtroxDev) [inline resources/views/app.blade.php]: shouldn't this be: ```{{ asset('css/app.css') }}```

Comment (GrahamCampbell) [inline resources/views/app.blade.php]: Doesn't matter.

Comment (AtroxDev) [inline resources/views/app.blade.php]: Yeah but isn't it cleaner without URL::? I'm just asking :)

**Comment by taylorotwell** [comment]
Laravel is meant to run in your document root as specified by your web server. I've never really liked the whole sub-folder business.

**Comment by franzliedke** [comment]
@taylorotwell But why? It's really easy to do, and having an app in a subfolder is a legitimate, and wide-spread use case.

**Pull request closed without merging**